src/commands/lodestone.ts

```
@@ -1,239 +1,477 @@
+          .map(([key, data]) => {
+            if (typeof data === "object" && data !== null && "level" in data) {
+              return `${key}: Level ${data.level}`;
+            }
+            return `${key}: Unknown`;
+          })
+          .join("\n"),
+        inline: false,
+      });
+    }
+
+    if (info.bozja && info.bozja.level) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)